src/mailing/sender.js

```diff
@@ -31,4 +34,34 @@ async function sendTestEmail(recipient) {
   });
 }
 
-module.exports = sendTestEmail;
+async function sendForgotPasswordEmail(toEmail, resetToken) {
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: process.env.EMAIL_USER,
+      pass: process.env.EMAIL_PASSWORD,
+    },
+  });
+
```

Comment:
We don't need to redefine the transporter each time this function is called and instead should use the one defined at the top level of this module/file (line 7).

Reply:
As far as I understand we want to delete this sender.js in email folder

Reply:
Yes we'll delete the `email/sender.js`. Here in the `mailing/sender.js` we can use the outlook `transport` variable defined earlier in the file to send an email in this function from the [`codethedream.practicum.team4@outlook.com`](mailto:codethedream.practicum.team4@outlook.com) email rather than defining a new gmail transporter (unless that's needed for a particular reason).

Reply:
email/sender.js deleted 62dcc9e089bdc1bb02458a59be648574de39b1f7